cmd/stop.go

```diff
@@ -26,6 +26,8 @@ var stopCmd = &cobra.Command{
 	Short: "Stop a task",
 	Long:  `Stops working on a task and updates the end time.`,
 	Run: func(cmd *cobra.Command, args []string) {
+		// Try to set the root persistence directory.
+		clerk.SetRootPersistenceDir(rootPersistenceDir)
```

Comment:
Does cobra automatically check if rootPersistenceDir is not set and do nothing here if it is?

Reply:
It just sets it to an empty string. This is an override, so it does nothing if the string is empty.